backend/core/domains/payments/services/payment_event_processor.py

```
@@ -0,0 +1,480 @@
+                    trigger_type='PAYMENT_RECEIVED',
+                    data={
+                        'payment_id': payment.id,
+                        'payment_number': payment.payment_number,
+                        'amount': str(payment.amount),
+                        'currency': payment.currency
+                    }
+                )
+
+                logger.debug(f"Triggered workflow advancement for payment {payment.payment_number}")
+
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)